app/bundles/CampaignBundle/Translations/en_US/messages.ini

```diff
@@ -225,3 +225,14 @@ mautic.config.peak_interaction_timer.fetch_interactions_from.tooltip='Select the
 mautic.config.peak_interaction_timer.fetch_limit.tooltip='Set the maximum number of interactions of each type (e.g., page hits, email reads) to retrieve for timing optimization.'
 mautic.campaign.parent.details='The event followed the <strong>%path%</strong> path of the %type% "<strong>%name%</strong>". Here are the details:'
 mautic.campaign.event.path.tooltip='This event followed the %path% path.'
+mautic.campaign.timeline.user_name="User Name"
+mautic.campaign.timeline.event_type="Action"
```

Comment:
Can we use `mautic.campaign.event.action.header` or `mautic.audit_log.report.action` instead of adding new key with the same word? It would save some work for the translators.